app/views/shopping_list_items/edit.html.erb

```diff
@@ -1,10 +1,11 @@
-<h3>
-  <%= link_to "", @shopping_list, class: Icon.back %>
-  <%= @shopping_list.name %>
-</h3>
+<p><%= link_to MaterialIcon.new(icon: :arrow_left).render + "#{@shopping_list.name} List", @shopping_list %></p>
+
 <h1>Editing <%= @shopping_list_item.name %></h1>
 
 <%= render 'form' %>
 
-<hr>
-<%= link_to "Delete", shopping_list_shopping_list_item_path(@shopping_list_item.list, @shopping_list_item), method: :delete, data: { confirm: "Are you sure you want to delete #{@shopping_list_item.name}? This action cannot be undone." }, class: button_classes('danger') %>
+<footer class='footer--delete'>
```

Comment:
This one could not use the partial as easily as the others, so i copy/pasted and moved along.